src/main/java/com/checkmarx/ast/wrapper/CxWrapper.java

```diff
@@ -263,6 +263,10 @@ public Results results(@NonNull UUID scanId) throws IOException, InterruptedExce
 
     public String results(@NonNull UUID scanId, ReportFormat reportFormat)
             throws IOException, InterruptedException, CxException {
+        return results(scanId, reportFormat, null);
+    }
+    public String results(@NonNull UUID scanId, ReportFormat reportFormat, String agent)
```

Comment:
this is a private method which means the plugin will not see the new contract. I'm not sure it's the right thing to do.